### DISCUSSION TITLE
Configured URL not showing on internet radio directories

### DISCUSSION THREAD

**Discussion by BornKillaz** [AzuraCast/AzuraCast#7332] [Q&A]
Hello.

Can anybody explain me how to properly configure station URL  to be shown in the various radio directories, like https://www.internet-radio.com/ and https://dir.xiph.org/

I have configured the URL where you configure genre and station name, but the URL simple does not show up on these directories.

Thank you.

**Comment by gunsar** [comment]
> Hello.
> 
> Can anybody explain me how to properly configure station URL to be shown in the various radio directories, like https://www.internet-radio.com/ and https://dir.xiph.org/
> 
> I have configured the URL where you configure genre and station name, but the URL simple does not show up on these directories.
> 
> Thank you.

You can try editing the mount point and adding it as shown in the picture.
![edit-moutpoint](https://github.com/user-attachments/assets/75c15c01-3fb2-4090-8fd3-b43a5ee910ce)

**Reply by atmix** [reply]
Unfortunately, this does not help

**Reply by gunsar** [reply]
> @atmix Unfortunately, this does not help

look at the logs -> Icecast Error Log
you will see the problem there, and remember not all servers/vps that we have can send data to the directory internet-radio/icecast directory, there are many possibilities such as server rejection, duplicate etc.

**Reply by atmix** [reply]
There are such entries in the logs:
yp/destroy_yp_server active ypdata not freed up
yp/yp_process_server error detected, backing off on http://icecast-yp.internet-radio.com
yp/send_to_yp connection to http://icecast-yp.internet-radio.com failed on /live with "Resolving timed out after 3000 milliseconds"

**Comment by atmix** [comment]
Maybe it's because of the Referrer-Policy: no-referrer-when-downgrade header, but I'm not sure)

**Comment by paulcjkp** [comment]
https://www.internet-radio.com/community/threads/icecast-server-yp-directory-settings.22223/

Go to Edit Profile, Broadcasting, and at the very bottom is Custom Configuration.  Follow the instructions on the above link.  Also, they do require that your site has a link back to them, make sure that's setup, or they will delete your station from their directory.

**Reply by atmix** [reply]
This requirement is only for shoutcast stations.
